core/src/cluster_slots.rs

```diff
@@ -172,16 +172,18 @@ impl ClusterSlots {
         if repair_peers.is_empty() {
             return Vec::default();
         }
+        let mut max_stake = 0;
         let stakes = {
             let validator_stakes = self.validator_stakes.read().unwrap();
             repair_peers
                 .iter()
                 .map(|peer| {
-                    validator_stakes
+                    let peer_stake = validator_stakes
                         .get(&peer.id)
                         .map(|node| node.total_stake)
-                        .unwrap_or(0)
-                        + 1
```

Comment:
I think you want to preserve this `+1` either here or in the normalized value below.

Reply:
ick, bad copy pasta, thanks!

Reply:
looks like a commit adds `+1` but the next one removes it.